meta-genivi-dev/recipes-dev-platform/packagegroups/packagegroup-gdp-sdl.bb

```diff
@@ -0,0 +1,18 @@
+DESCRIPTION = "Smart Device Link (SDL) package group"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${IVI_COREBASE}/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+# Avoid hardcoding the full layer path into the checksums
```

Comment:
This packagegroup bb file is inherited from other packagegroup in meta-genivi-dev/recipes-dev-platform/packagegroups/. This is not FIXME. It 's a comment that is introduced in [ccad2e3](https://github.com/GENIVI/meta-ivi/commit/ccad2e3) .